src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -207,26 +217,28 @@ const props = withDefaults(defineProps<{
 	 */
 	confirm?: boolean
 
-	/**
-	 * Default increment step for minutes in the time picker.
-	 * @default 10
-	 */
-	minuteStep?: number
-
 	/**
 	 * Preview format for the picker input field.
+	 * Can either be a string of Unicode tokens or a function that takes a date,
+	 * or for range picker an array of two dates, and returns the formatted date as string.
 	 *
 	 * @default Intl.DateTimeFormat is used to format dates and times
 	 * @see https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table
 	 */
-	format?: string
+	format?: string | ((date: Date) => string) | ((dates: Date[]) => string)
```

Comment:
Tuples aren't a bad practice, as Enums, or I'm missing something? 
```suggestion
	format?: string | ((date: Date) => string) | ((dates: [Date, Date]) => string)
```

Reply:
No, tuple is a type and enum is not (not only)

Reply:
Tuple are just more restrictive array types, in this case it is exactly two Date objects.
(Which is not true without the outher PR as for partial range it could have emitted `[Date, null]` but we fixed that :heavy_check_mark: )